test/core/parser/test_parser.py

```diff
@@ -694,10 +694,14 @@ def testFraction(self):
         self.check("\\(   \\/   \\)", 'FractionBox["", ""]')
 
     def testFormBox(self):
-        self.check("\\( 1 \\` b \\)", 'FormBox["b", Removed["$$Failure"]]')
         self.check("\\( \\` b \\)", 'FormBox["b", StandardForm]')
         self.check("\\( a \\` b \\)", 'FormBox["b", a]')
         self.check("\\( a \\` \\)", 'FormBox["", a]')
+        self.check("\\( a \\` b + c \\)", 'FormBox[RowBox[{"b", "+", "c"}], a]')
```

Comment:
A small thing but I would appreciate it if you' use raw strings here as well. Thanks.

Reply:
I just followed the convention used in this file. But I agree, it would be clearer with raw strings.

Reply:
Done in [f6f3d1f](https://github.com/Mathics3/mathics-core/pull/545/commits/f6f3d1fc29ed338d92829377f77d6a521e099129)